jmix-ui/ui/src/main/java/io/jmix/ui/component/filter/FilterSupport.java

```diff
@@ -82,10 +82,10 @@ public void removeCurrentFilterConfiguration(Filter filter) {
 
     public ScreenFragment createFilterConfigurationFragment(FrameOwner owner,
                                                             boolean isNewConfiguration,
-                                                            Filter.Configuration currentConfiguration) {
+                                                            Filter filter) {
         Fragments fragments = UiControllerUtils.getScreenContext(owner).getFragments();
         FilterConfigurationModelFragment fragment = fragments.create(owner, FilterConfigurationModelFragment.class);
-        initFilterConfigurationFragment(fragment, isNewConfiguration, currentConfiguration);
+        initFilterConfigurationFragment(fragment, isNewConfiguration, filter.getCurrentConfiguration());

```

Comment:
What the point of this change in public API (even though it's an internal bean) if you still get `CurrentConfiguration` from passed filter.